tests/www/api/experimental/test_endpoints.py

```diff
@@ -291,14 +292,10 @@ class TestPoolApiExperimental(TestBase):
     @classmethod
     def setUpClass(cls):
         super(TestPoolApiExperimental, cls).setUpClass()
-        session = Session()
-        session.query(Pool).delete()
-        session.commit()
-        session.close()
 
     def setUp(self):
         super().setUp()
-
+        clear_db_pools(create_default_pool=False)
```

Comment:
I think it would be better to update the tests here to expect the default pool to exist - and it shouldn't be possible to Airflow to get into a state where the default_pool doesn't exist, so we might end up testing an "impossible" case.

Reply:
Yeah just updated.